### PR TITLE
adding info about iCub CPUs

### PR DESCRIPTION
**Pull request by maggia80**
all possible CPU boards installed on iCub robot

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Thanks heaps @maggia80 for this addition! 

I think it'd be a bit cleaner to publish this information in a dedicated file, instead.

File: docs/icub_operating_systems/index.md [hunk 1]
```diff
@@ -18,3 +18,34 @@ Once you customized your machine, you can proceed with further setup for special
...
+
+<center> <img src ="../img/congatec.png" width=300>       
+</center>
+
+CONGATEC
+
+<center> <img src ="../img/kontron.png" width=300>       
+</center>
+
+KONTRON
```
Comment (pattacini) [inline docs/icub_operating_systems/index.md:51]: Also, you may want to use the following format:

```
<center>
  <figure>
    <img src="../img/filename"/>
    <figcaption><b>Figure Caption</b></figcaption>
  </figure>
</center>
```

**Comment by mbrunettini** [comment]
I Agree with @pattacini : I'd rather put the above info in a dedicated page under "iCub HW" section rather then under "iCub OS", then we can add a link from the "The OS on the iCub Head" section there

File: docs/icub_operating_systems/index.md (hunk 1 above)
Comment (pattacini) [reply docs/icub_operating_systems/index.md:51]: Don't forget to push the image files as well.

**Comment by maggia80** [comment]
> I Agree with @pattacini : I'd put the above info in a dedicated page under "iCub HW" section rather then under "iCub OS", then we can add a link from the "The OS on the iCub Head" section there

You are right, it's HW-related but it reports info that are more useful to people installing the OS/SW rather than HW guys. That's why I thought it could be placed here. I could remove the imgs because they are quite useless.

If you agree we can leave the table here until there will be a proper page for the iCub Head with all the HW specs.

**Comment by mbrunettini** [comment]
I prefer to have a link in this section to  separate page under "HW" with this table. 
To have here is really misleading

**Comment by mbrunettini** [comment]
I can propose a PR with the modifications I meant

**Comment by pattacini** [comment]
> I can propose a PR with the modifications I meant

Hold on, @maggia80 is on this.

File: docs/icub_operating_systems/index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This manual describes how to install and customize the linux operating systems r
 
 ## Robot OS
 
-There are two versions of the customized OS that runs on the machine inside iCub head:
+There are two versions of the customized OS that runs on the machine inside iCub head*:
```
Comment (pattacini) [inline docs/icub_operating_systems/index.md:7]:
```suggestion
There are two versions of the customized OS that run on the machine inside iCub head.

!!! note
    For further details, please refer to the [iCub CPU boards](../icub_cpu_boards/icub_cpu_boards.md)
```

File: docs/icub_operating_systems/index.md [hunk 3]
```diff
@@ -18,3 +18,6 @@ Once you customized your machine, you can proceed with further setup for special
 - [_CUDA workstation_](./other-machines/cuda-workstation.md) - a GPU-equipped machine with CUDA software installed
 - [_The iCub Server_](./other-machines/icub-server-from-scratch.md) - a machine acting only as server (for Network gateway, NTP, NFS, DHCP and DNS services)
 - [_The iCub Laptop Server_](./other-machines/icub-server-laptop.md) - a single machine, usually a laptop, acting both as server (for Network Gateway, ntp and NFS serviceas) as well as robot console
+
+
+* for details please refer to [iCub CPU boards](./icub_cpu_boards/icub_cpu_boards.md)
```
Comment (pattacini) [inline docs/icub_operating_systems/index.md:23]:
```suggestion

```

**Review by pattacini — APPROVED** [review]
Minor modifications are done straight as proposed changes.

File: docs/icub_operating_systems/index.md [hunk 4]
```diff
@@ -18,3 +21,6 @@ Once you customized your machine, you can proceed with further setup for special
...
```
Comment (pattacini) [inline docs/icub_operating_systems/index.md:26]:
```suggestion

```

**Comment by mbrunettini** [comment]
Fine! thanks

**Pull request merged**